Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -12329,7 +12329,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Unknown Author"
+            "value" : "unknown"
```

Comment:
I changed this because our users don't know what an "Author" is. It is used when we don't know the display name of someone who made a report, so it would now read something like "This has been reported by **unknown** for spam."